faker/providers/misc/__init__.py

```diff
@@ -613,6 +614,28 @@ def create_json_structure(data_columns: Union[Dict, List]) -> dict:
         data = [create_json_structure(data_columns) for _ in range(num_rows)]
         return json.dumps(data, indent=indent, cls=cls)
 
+    def xml(self,
+        nb_elements: int = 10,
+        variable_nb_elements: bool = True,
+        value_types: Optional[TypesSpec] = None,
+        allowed_types: Optional[TypesSpec] = None,
+        ) -> str:
+        """
+        Returns a xml.
+
+        :nb_elements: number of elements for dictionary
+        :variable_nb_elements: is use variable number of elements for dictionary
+        :value_types: type of dictionary values
+        """
+        try:
+            import xmltodict
+        except ImportError:
+            raise UnsupportedFeature("`xml` requires the `xmltodict` python library.", "xml")
```

Comment:
In order to run the test, you'll need to add `xmltodict` to the [`tox.ini`](https://github.com/joke2k/faker/blob/master/tox.ini#L6-L13) file